packages/amazonq/src/lsp/chat/webviewProvider.ts

```diff
@@ -144,7 +146,7 @@ export class AmazonQChatViewProvider implements WebviewViewProvider {
                     const vscodeApi = acquireVsCodeApi()
                     const hybridChatConnector = new HybridChatAdapter(${(await AuthUtil.instance.getChatAuthState()).amazonQ === 'connected'},${featureConfigData},${welcomeCount},${disclaimerAcknowledged},${regionProfileString},${disabledCommands},${isSMUS},${isSM},vscodeApi.postMessage)
                     const commands = [hybridChatConnector.initialQuickActions[0]]
-                    amazonQChat.createChat(vscodeApi, {disclaimerAcknowledged: ${disclaimerAcknowledged}, quickActionCommands: commands}, hybridChatConnector);
+                    amazonQChat.createChat(vscodeApi, {disclaimerAcknowledged: ${disclaimerAcknowledged}, pairProgrammingAcknowledged: ${pairProgrammingAcknowledged}, quickActionCommands: commands}, hybridChatConnector);
```

Comment:
Is this line temporary for hybrid chat. Just wondering about scalability as we add more props to it

Reply:
what's passed in here is synced with what the flare chat client expects https://github.com/aws/language-servers/pull/1023/files#diff-b68ed2121e608d8cb0c35bb3021bfdc77a4fc0bd423d7b79cbead1e290c15300R135